models/boards.js

```diff
@@ -334,6 +394,18 @@ Boards.helpers({
       swimlane.copy(_id);
     });
   },
+  /**
+   * Generates random chars as board key
+   */
+  generateKey() {
+    const chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
This function should check for the existence of the key. It should unique with regards to the rest of the keys in the system. With only 3 digits here, you're going to run out of unique keys very quickly. Additionally, this should default to the first 3 characters of the `title` being created for the board.

Example:

You have a board called `Test`

You create a board called `Test`

The first board should have a board key `TES` and the second one should have a board key `TEST`

You create another board called `Test`. It should have a board key called `TEST2`. Basically, you want to make sure the keys are unique and that they (hopefully) follow the name of the board as closely as possible.